crates/block2/src/stack.rs

```diff
@@ -190,6 +189,9 @@ where
     /// encoding information string themselves, thus obtaining a block
     /// containing it and working with these APIs.
     ///
+    /// You provide the encoding through the `E` type parameter, which shoul
```

Comment:
`s/shoul/should/` :grin:

Reply:
Ooops, fixed in 3d6ba5e246af8982e08ff0274435164756c8241e